sap-s4hana-soap/2.2/modules/ROOT/pages/sap-s4hana-soap-connector-reference.adoc

```diff
@@ -0,0 +1,558 @@
+= SAP S/4HANA SOAP Connector 2.2 Reference
+:page-aliases: connectors::sap/sap-s4hana-soap-connector-reference.adoc
+
+Support Category: https://www.mulesoft.com/legal/versioning-back-support-policy#anypoint-connectors[Premium]
+
+Anypoint Connector for SAP S/4HANA SOAP provides full support for the SOAP APIs exposed by SAP S/4HANA ERP and from version 2.1.x provides support also for SAP S/4HANA On-Premise ERP.
```

Comment:
This sentence reads funny. I would say this: "Anypoint Connector for SAP S/4HANA SOAP provides full support for the SOAP APIs exposed by SAP S/4HANA ERP and, starting with 2.1.x, provides support also for SAP S/4HANA On-Premise ERP.